docs/troubleshooting-guide/filters.md

```diff
@@ -1,72 +1,90 @@
 ---
-title: My dashboard filters don't work
+title: Troubleshooting filters
 ---
 
-# My dashboard filters don't work
 
-You've tried to add a [filter widget][filter-widget-gloss] to your dashboard, but:
+# Troubleshooting filters
 
-- the question you want to connect the filter to doesn't show up, or
-- the filter doesn't show a dropdown list of possible values when you use it, or
-- the filter removes all of the rows from the table no matter what value you set it to.
+It's always a good idea to start with a quick sanity check:
 
-If you've created a [linked filter][linked-filter-gloss], please see [this troubleshooting guide][troubleshoot-linked-filters] instead.
+1. Clear your browser cache.
+2. Refresh the page.
+3. Open your question or dashboard in an incognito window.
 
-If you're using a [SQL variable][sql-variable-gloss] with the variable type "field filter", go to [Troubleshooting SQL variables][troubleshoot-sql-variables].
+## Dashboard filters
 
-## Is the dashboard filter actually connected to your question?
+1. Go to your dashboard.
+2. Click the **pencil** icon to go into edit mode.
+3. Click the **gear** icon beside your filter widget.
+4. Make sure you've selected a column for your filter under **Column to filter on**.
+5. If you can't find the right **Column to filter on**, or you're getting "No Results" when you apply the filter:
+   - Go to the _original question_.
+   - Follow the troubleshooting steps under [Question filters](#question-filters).
 
-**Root cause:** The filter isn't connected to any cards on the dashboard, or connected to the wrong field.
+## Question filters
 
-**Steps to take:**
+1. Go to your question.
```

Comment:
> 1. Go to your question.

Do we need this step? If so, we should be more specific on how to get from dashboard to question.